devops/demo/landing-page/webroot/index.html

```diff
@@ -24,52 +32,70 @@ <h1>Demo Server</h1>
+                    </td>
                 </tr>
 
                 <tr>
                     <th>Passphrase</th>
-                    <td>correct horse battery staple profanity oil chewy</td>
-                </tr>
-
-                <tr>
-                    <th>TOTP secret</th>
-                    <td>JHCOGO7VCER3EJ4L</td>
+                    <td><span id="passphrasevalue">correct horse battery staple profanity oil chewy</span>
+                        <span class="copy-btn" data-target="#passphrasevalue"><svg width="18" height="18"><use href="#copy-icon"></use></svg></span>
+                    </td>
                 </tr>
 
                 <tr>
                     <th>Current TOTP token</th>
                     <td id="totp">
                         <span id="totpvalue">......</span>
-                        <!-- Apache-licensed "content copy" icon from https://material.io/resources/icons/ -->
-                        <span id="totpcopy"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="18px" height="18px"><path d="M0 0h24v24H0z" fill="none"/><path d="M16 1H4c-1.1 0-2 .9-2 2v14h2V3h12V1zm3 4H8c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h11c1.1 0 2-.9 2-2V7c0-1.1-.9-2-2-2zm0 16H8V7h11v14z"/></svg></span>
+                        <span class="copy-btn" data-target="#totpvalue"><svg width="18" height="18"><use href="#copy-icon"></use></svg></span>
                         <span id="totpttl"></span>
                     </td>
                 </tr>
+
+                <tr>
+                    <th>TOTP secret</th>
+                    <td><span id="totpsecretvalue">JHCOGO7VCER3EJ4L</span>
+                        <span class="copy-btn" data-target="#totpsecretvalue"><svg width="18" height="18"><use href="#copy-icon"></use></svg></span>
+                    </td>
+                </tr>
+
             </tbody>
         </table>
         <script src="/assets/js/jsOTP.js"></script>
         <script>
-            document.getElementById("totpcopy").addEventListener(
-                "click",
-                function(e) {
-                    navigator.clipboard.writeText(
-                        document.getElementById("totpvalue").textContent
-                    );
-                }
-            )
+            // Add "copy to clipboard" functionality to all copy buttons
+            document.querySelectorAll(".copy-btn").forEach(btn => {
+                btn.addEventListener("click", () => {
+                    const selector = btn.dataset.target;
+                    const text = document.querySelector(selector).textContent.trim();
+                    navigator.clipboard.writeText(text);
+
+                    // Display exactly one ephemeral success message
+                    const cell = btn.parentNode;
+                    const existing = cell.querySelector(".copiedtip");
+                    if (existing) existing.remove();
```

Comment:
Please add braces, let's not have a goto fail here :)
```suggestion
                    if (existing) {
                        existing.remove()
                    };
```

Reply:
Aw, I've always liked that style. :P Done.